### PR TITLE
suthesh/fix live chat email auto population on login

### PR DESCRIPTION
**Pull request by suthesan-deriv**
# Description

Auto populate email field of client upon login for CS team 

Changes:

-   Auto populate email field of client upon login for CS team 

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by mustofa-binary — APPROVED** [review]

**Review by mustofa-binary — CHANGES_REQUESTED** [review]
test is failing

File: src/components/layout/livechat.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import Cookies from 'js-cookie'
 import styled from 'styled-components'
+import * as CustomerSDK from '@livechat/customer-sdk'
```
Comment (mustofa-binary) [inline src/components/layout/livechat.js:5]: can we only import the function we need 

```suggestion
import { init } from '@livechat/customer-sdk'
```

File: src/components/layout/livechat.js [hunk 2]
```diff
@@ -38,79 +49,123 @@ const LiveChat = ({ LC_API, is_livechat_interactive, setLiveChatInteractive }) =
         if (callback) callback()
     }
 
+    const checkCookie = function () {
```
Comment (mustofa-binary) [inline src/components/layout/livechat.js:52]: we can use arrow anonymous function here

File: src/components/layout/livechat.js [hunk 3]
```diff
@@ -38,79 +49,123 @@ const LiveChat = ({ LC_API, is_livechat_interactive, setLiveChatInteractive }) =
...
+                                `${first_name} ${last_name}`,
+                            )
+                        }
+                    } else {
+                        if (window.LiveChatWidget.get('chat_data')) {
+                            const chatID = window.LiveChatWidget.get('chat_data').chatId;
+                            customerSDK
+                                .deactivateChat({ chatId: chatID })
+                                .then(response => {
+                                    console.log(response) //eslint-disable-line
```
Comment (mustofa-binary) [inline src/components/layout/livechat.js:154]: console log 😅

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: src/components/layout/livechat.js [hunk 4]
```diff
@@ -26,6 +27,16 @@ const StyledLiveChat = styled.div`
...
     const [is_livechat_hover, setLivechatHover] = React.useState(false)
+    const [is_logged_in, setLoggedIn] = React.useState(false)
+    const domain = (window.location.hostname.includes('deriv.com')
+        ? 'deriv.com'
+        : 'binary.sx'
+    )
+
+    const customerSDK = CustomerSDK.init({
+        licenseId: 12049137,
+        clientId: '66aa088aad5a414484c1fd1fa8a5ace7',
```
Comment (mustofa-binary) [inline src/components/layout/livechat.js:38]: can store these on utility

**Review by kevinw-binary — CHANGES_REQUESTED** [review]
Please help resolve confilcts

**Review by mustofa-binary — APPROVED** [review]

**Review by kevinw-binary — APPROVED** [review]

**Pull request merged**